.changeset/olive-llamas-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
```

Comment:
it's also LLM patch

Reply:
done and added also common with the refactor :)